driver/razermouse_driver.h

```diff
@@ -56,6 +56,8 @@
 #define USB_DEVICE_ID_RAZER_MAMBA_WIRELESS_WIRED 0x0073
 #define USB_DEVICE_ID_RAZER_PRO_CLICK_RECEIVER 0x0077
 #define USB_DEVICE_ID_RAZER_VIPER 0x0078
+#define USB_DEVICE_ID_RAZER_VIPER_MINI_SE_WIRED 0x009E
+#define USB_DEVICE_ID_RAZER_VIPER_MINI_SE_WIRELESS 0x009F
```

Comment:
please keep the list sorted by PID, so place below `USB_DEVICE_ID_RAZER_DEATHADDER_V2_X_HYPERSPEED`

Reply:
Sorted. Sorry about that.